elasticsearch5/src/main/resources/log4j2.properties

```diff
@@ -0,0 +1,7 @@
+appender.console.type = Console
```

Comment:
It's for any internal logging produced by the transport client (for example, if there are issues connecting to the Elasticsearch cluster, a user can set `es.setting.logger.org.elasticsearch` to `debug` and see the debugging logging by the transport client to help debug such an issue. I pushed a65dcb76d651cfe83bf1a2edf0671afaf5cc3142 so that these logs go to stderr.